### PR TITLE
Gives the mercs a nuke...

### PR DESCRIPTION
**Pull request by Mark9013100**
beer keg

### CODE REVIEW

**Comment by Yoshax** [comment]
No flashing nuke lights :-1:

**Comment by Mark9013100** [comment]
I'm a lazy asshole and that'd take effort

File: code/modules/reagents/reagent_dispenser.dm [hunk 1]
```diff
@@ -217,6 +217,16 @@
...
 
+/obj/structure/reagent_dispensers/beerkeg/fakenuke
+	name = "nuclear beer keg"
+	desc = "A beer keg in the form of a nuclear bomb! An absolute blast at parties!"
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "nuclearbomb0"
+	amount_per_transfer_from_this = 10
+	New()
+		..()
+		reagents.add_reagent("beer",1000)
```
Comment (Kelenius) [inline code/modules/reagents/reagent_dispenser.dm]: This will add beer twice. It's a subtype of regular keg, it inherits the procs.

**Comment by Datraen** [comment]
What Kelenius said, + breaking current coding standards.

Beerkegs are older code, use relative pathing instead of absolute, which is the preferred method of pathing at the moment.
See https://github.com/PolarisSS13/Polaris/wiki/Code-Standards

**Comment by Hubblenaut** [comment]
> use relative pathing instead of absolute
Don't you mean the other way around?

**Comment by Datraen** [comment]
If you take that snippet, yes that is correct.
I'll write it out without the first comma:
Beerkegs are older code that use relative pathing instead of absolute pathing, which is the preferred method.

It probably reads better without the first comma, with less of a chance for miscommunication. We use absolute pathing, and not relative.

**Comment by Neerti** [comment]
Pathing issue was fixed, so :+1:

**Pull request merged**